Sprint-2/interpret/invert.test.js

```diff
@@ -0,0 +1,12 @@
+const invert = require("./invert.js");
+// E.g. invert({x : 10, y : 20}), target output: {"10": "x", "20": "y"}
+describe("invert", () =>{
+    [
+        {input:{x : 10, y : 20}, expected:{"10": "x", "20": "y"}},
+        {input:{ a : 1 }, expected:{ 1 : "a" }},
+        {input:{a : 1, b: 2}, expected:{ 1 : "a" , 2 : "b" }},
```

Comment:
what happens if there are multiple identical values? Objects cannot have multiple identical keys. 

I'm not sure what should happen in this case.

You could test for it with 
 ```suggestion
        {input:{a : 1, b: 1}, expected:/* What happens here? */ },
```